includes/functions/llms.functions.template.php

```diff
@@ -5,17 +5,18 @@
  * @package LifterLMS/Functions
  *
  * @since unknown
- * @version unknown
+ * @version [version]
  */
 
 defined( 'ABSPATH' ) || exit;
 
 /**
  * Get template part
  *
- * @param  string $slug [url slug of template]
- * @param  string $name [name of template]
+ * @since unknown
```

Comment:
The "unknown" version tag should be capitalized because I'm anal.

```suggestion
 * @since Unknown
```